.travis.yml

```diff
@@ -4,21 +4,14 @@ dist: trusty
 group: edge
 
 before_install:
```

Comment:
You also need libomp-dev if you compile sleef with clang.
Also note that clang version 3.9 and later are only supported.